pinot-spi/src/main/java/org/apache/pinot/spi/config/ConfigUtils.java

```diff
@@ -31,6 +31,9 @@
 public class ConfigUtils {
   private static final Map<String, String> ENVIRONMENT_VARIABLES = System.getenv();
 
+  private ConfigUtils() {
```

Comment:
(nit) We usually put this right next to the class definition for slightly better readability, see other utils class for example. Same for other util classes